cli-client/recognizer_stream.py

```diff
@@ -105,7 +104,7 @@ def _response_watcher(
             for response in response_iterator:
                 json = MessageToJson(response)
                 logging.info("New incoming response: '%s ...'", json[0:50].replace('\n', ''))
-                print(MessageToJson(response))
+                logging.debug(MessageToJson(response))
```

Comment:
@mjbverbio this print here was intented, it was for the result to be printed in stdout insted of stderr, so that we could do things like: 

`./recognizer_stream.py -a audio.wav  -T GENERIC -l en-US -t token -s 8000 | jq ".result.alternatives[0].transcript"`
or 
`./recognizer_stream.py -a audio.wav  -T GENERIC -l en-US -t token -s 8000 > result.json`

Do you think it would be useful to leave it as a print instead of a call to logging.debug ?